src/js/imageEditor.js

```diff
@@ -486,6 +486,16 @@ class ImageEditor {
         this._graphics.renderAll();
     }
 
+    /**
+    * set selection style properties
+    * @param {Object} styles - Object of styles
+    * @example
+    * imageEditor.setSelectionStyle({ cornerSize: 50 });
+    */
+    setSelectionStyle(styles) {
+        this._graphics.setSelectionStyle(styles);
+    }
+
```

Comment:
There is an option for setting `selectionStyle`.

I don't think this method is required.

Reply:
@junghwan-park indeed there is but only if you include ui. If you don't include UI. then there is no way to change selectionStyles i.e. cornerSize.
Hence one can use this method to do something like:


```
imageEditor.setSelectionStyle({ cornerSize: 10 });
```